File: Data Structure/Lab 3/Replace.c
// Read n numbers in an array then read two different numbers, replace 1st 
//number with 2nd number in an array and print its index and final array.

#include<stdio.h>
void main(){
    int n,arr[n],i = 0,n1,n2,index =0;

    printf("Enter size of an array(n) :");
    scanf("%d",&n);

    printf("Enter n1:");
    scanf("%d",&n1);

    printf("Enter n2:");
    scanf("%d",&n2);

    for(i = 0; i < n ; i++){
        printf("Enter element :");
        scanf("%d",&arr[i]);
    }

    for(i = 0 ; i < n; i++){
        if(n1 == arr[i]){
            arr[i] = n2;
            index = i; 
        }
    }

    for(i = 0; i < n ; i++){
        printf("%d ,",arr[i]);
    }
}

File: Data Structure(Java)/Lab 2/Replace.java
// Read n numbers in an array then read two different numbers, replace 1st 
//number with 2nd number in an array and print its index and final array.

import java.util.Scanner;

public class Replace{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter size of an array :");
        int n = sc.nextInt();

        int[] arr = new int[n];

        System.out.print("Enter number 1 :");
        int n1 = sc.nextInt();

        System.out.print("Enter number 2 :");
        int n2 = sc.nextInt();
        int index = 0;

        for(int i = 0 ; i < arr.length; i++){
            System.out.println("Enter Element at "+i);
            arr[i] = sc.nextInt();
        }
        for(int i = 0 ; i < arr.length; i++){
            if(n1 == arr[i]){
                arr[i] = n2;
                index = i;
            }
        }

        System.out.println("Final Array :");
        for(int i = 0; i < arr.length ; i++){
            if(i == arr.length - 1){
                System.out.print(arr[i]);
            }
            else{
                System.out.print(arr[i]+",");
            }
        }
        System.out.println("Index of replacment :"+index);

        sc.close();
    }
}